saleor/graphql/shipping/tests/test_shipping_zones_query.py

```diff
@@ -390,3 +405,26 @@ def test_shipping_method_tax_class_query_by_staff(
     content = get_graphql_content(response)
     data = content["data"]
     assert data["shippingZone"]["shippingMethods"][0]["taxClass"]["id"]
+
+
+def test_shipping_method_update_no_countries_in_data(
```

Comment:
This test should be in `test_shipping_zone_update.py` not in `test_shipping_zones_query.py`.